opts/secret.go

```diff
@@ -50,7 +50,7 @@ func (o *SecretOpt) Set(value string) error {
 
 		value := parts[1]
 		switch key {
-		case "source":
+		case "source", "src":
 			options.Source = value
 		case "target":
```

Comment:
Don't we need "dst" and "destination" for consistency with `--mount`?
https://github.com/docker/docker/blob/5e143b5f8d82537635a57f3d9eda8126caf68a6f/opts/mount.go